docs_app/content/tutorial/5-implementing-operators.md

```diff
@@ -0,0 +1,103 @@
+# 5. Implementing Operators
```

Comment:
Advanced: This should be moved to a more "Advanced" section. IMHO, the information presented here is not part of the "Essential" basic knowledge required to use RxJS successfully.

The majority of RxJS users will not have to implement their own operators. For some developers, even if they don't implement their own, understanding how it works on the inside would be helpful. But for those users with weaker TypeScript skills, this will be overwhelming.